tests/test_views.py

```diff
@@ -197,6 +199,21 @@ def get_queryset(self):
 
         TestView.as_view()(build_request())
 
+    def test_get_tables_class(self):
+        view = SimpleView()
+        table_class = view.get_table_class()
+        self.assertEqual(table_class, view.table_class)
+
+    def test_get_tables_class_auto(self):
+        view = SimpleNoTableClassView()
```

Comment:
I prefer having the definition of tables/views in the test as much as possible, so if only used in one test, please put SimpleNoTableClassView in this method.